src/components/settings-row/settings-row.component.js

```diff
@@ -17,6 +17,7 @@ const marginPropTypes = filterStyledSystemMarginProps(
 
 const SettingsRow = ({
   title,
+  headingLevel = 1,
```

Comment:
Personally I would amend this prop to accept strings in the form `h1`, `h2`, ... , `h6` so its consistent with the `Heading`'s `headingLevel` prop and `Typography`'s `variant` prop.

Reply:
I actually prefer the numeric `HeadingLevel` ironically - but I understand the argument about consistency with `Typography`, even though that really has to use strings because it has a much wider range of variants.

But I don't feel strongly so I haven't commented on this elsewhere in the review :-)